core/src/main/java/com/sequenceiq/cloudbreak/service/externaldatabase/DatabaseServerParameterSetter.java

```diff
@@ -0,0 +1,11 @@
+package com.sequenceiq.cloudbreak.service.externaldatabase;
+
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.cloudbreak.service.externaldatabase.model.DatabaseServerParameter;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.DatabaseServerV4StackRequest;
+
+public interface DatabaseServerParameterSetter {
```

Comment:
Could be named DatabaseServerParameterDecorator. Setter name is used for setter method, looks weird as class name,

Reply:
fine. renamed.